worlds/musedash/__init__.py

```diff
@@ -49,7 +49,7 @@ class MuseDashWorld(World):
     game = "Muse Dash"
     options_dataclass: ClassVar[Type[PerGameCommonOptions]] = MuseDashOptions
     topology_present = False
-    data_version = 10
+    data_version = 11
```

Comment:
would probably be better to just remove this since it's being removed soon anyways and has been deprecated for a while

Reply:
Done.